### DISCUSSION TITLE
Adaptive remeshing with time stepping

### DISCUSSION THREAD

**Discussion by maxxxzdn** [kinnala/scikit-fem#992] [Q&A]
Hi everyone, thanks for the great library! 
Do you know how one can implement adaptive remeshing for solving a PDE with a time derivative? I am interested in something like a combination of Example 19 and Example 22. 

Yours,
Max

**Comment by kinnala** [comment]
In general, there are several options. You could
1. Simply do adaptive refinement each time step and reuse the mesh from the previous time step
2. Do adaptive refinement each time step starting from the same base mesh over and over again
3. Reuse the mesh from the previous step and coarsen the mesh after refinement is not needed anymore

Coarsening is not supported right now in scikit-fem.
